File: DuDoong-Domain/src/main/java/band/gosrock/domain/domains/user/exception/AlreadyDeletedUserException.java
package band.gosrock.domain.domains.user.exception;


import band.gosrock.common.exception.DuDoongCodeException;
import band.gosrock.common.exception.GlobalErrorCode;

public class AlreadyDeletedUserException extends DuDoongCodeException {

    public static final DuDoongCodeException EXCEPTION = new AlreadyDeletedUserException();

    private AlreadyDeletedUserException() {
        super(GlobalErrorCode.USER_ALREADY_DELETED);
    }
}

File: DuDoong-Domain/src/main/kotlin/band/gosrock/domain/domains/user/exception/AlreadyDeletedUserException.kt
package band.gosrock.domain.domains.user.exception

import band.gosrock.common.exception.DuDoongCodeException

class AlreadyDeletedUserException private constructor() : DuDoongCodeException(UserErrorCode.USER_ALREADY_DELETED) {
    companion object {
        @JvmField
        val EXCEPTION: DuDoongCodeException = AlreadyDeletedUserException()
    }
}
